client/payment-request/index.js

```diff
@@ -311,34 +307,6 @@ jQuery( ( $ ) => {
 		 * @return {Object} Stripe paymentRequest element or custom button jQuery element.
 		 */
 		createPaymentRequestButton: ( elements, paymentRequest ) => {
-			let button;
-			if ( wcpayPaymentRequestParams.button.is_custom ) {
-				button = $( wcpayPaymentRequestParams.button.css_selector );
-				if ( button.length ) {
-					// We fallback to default paymentRequest button if no custom button is found in the UI.
-					// Add flag to be sure that created button is custom button rather than fallback element.
-					button.data( 'isCustom', true );
-					return button;
-				}
-			}
-
-			if ( wcpayPaymentRequestParams.button.is_branded ) {
-				if ( shouldUseGooglePayBrand() ) {
-					button = wcpayPaymentRequest.createGooglePayButton();
```

Comment:
The `createGooglePayButton` method doesn't seem to be used anywhere else anymore - can you please remove it?